msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AadInstanceDiscoveryProvider.java

```diff
@@ -47,7 +47,9 @@ class AadInstanceDiscoveryProvider {
 
         TRUSTED_HOSTS_SET.addAll(Arrays.asList(
                 "login.windows.net",
-                "login.microsoftonline.com"));
+                "login.microsoftonline.com",
+                "login.microsoft.com",
```

Comment:
Just had a look through the file's history to confirm, and it seems like that list has been the same since 2019 (I moved things around during the first region PR, but didn't add anything to it)

My best guess is that those URLs didn't exist at the time, and simply never got added.

Reply:
You are missing `login.windows.net`

https://login.microsoftonline.com/common/discovery/instance?api-version=1.1&authorization_endpoint=https%3A%2F%2Flogin.microsoftonline.com%2Fcommon%2Foauth2%2Fv2.0%2Fauthorize